k8s/src/main/scala/io/buoyant/k8s/Watchable.scala

```diff
@@ -167,10 +175,18 @@ private[k8s] abstract class Watchable[O <: KubeObject: TypeReference, W <: Watch
    * @return an [[Activity]]`[T]` updated by [[Watch]] events on this object,
    *         where `T` is the return type of the `onEvent` function
    */
```

Comment:
this comment probably needs updating, since the signature has changed significantly